c++/src/capnp/rpc.c++

```diff
@@ -3030,6 +3101,17 @@ private:
 
     // No more using `call` after this point, as it now belongs to the context.
 
+    if (call.getIsRealtime()) {
+      auto promiseAndPipeline = startCall(
+          call.getInterfaceId(), call.getMethodId(), kj::mv(capability), context->addRef(), hints);
+      tasks.add(promiseAndPipeline.promise
+          .then([](){}, [](kj::Exception&& exception) {
+            KJ_LOG(ERROR, exception);
+          })
+      );
+      return;
```

Comment:
@kentonv: does that seem correct? I have the feeling that this causes the segmentation fault I mention [in the comment above](https://github.com/capnproto/capnproto/pull/1460/files#r1610736509). Somehow it feels like I should be attaching the context around here before returning? Not sure.

Reply:
I think I was right in that I was missing the `.attach(kj::mv(context))` above, resulting in a segfault :+1: